### DISCUSSION TITLE
Is it possible to access docker containers using theirs private ip?

### DISCUSSION THREAD

**Discussion by joaoeduardo** [lima-vm/lima#610] [Q&A]
For example, running a nginx container with the command below:

```
~ docker run -p 8080:80 nginx
```

Then get the container IP with the following command:

```
~ docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' container_name
172.17.0.2
```

Is it possible to use this address to access the web page? Is there some config that make it possible?

Thanks!

Edit: I forgot to mention that I used the example [docker.yaml](https://github.com/lima-vm/lima/blob/master/examples/docker.yaml).

**Comment by AkihiroSuda** [comment]
This should work: https://norouter.io/docs/getting-started/vpn/
